src/engine/sys/sdl_input.cpp

```diff
@@ -1202,7 +1211,7 @@ void IN_Frame()
 		// Console is down in windowed mode
 		IN_SetFocus( false );
 	}
-	else if ( !( SDL_GetWindowFlags( window ) & SDL_WINDOW_INPUT_FOCUS ) )
+	else if ( com_unfocused->integer )
```

Comment:
(this one is for consistency)

Reply:
> [from there](https://github.com/DaemonEngine/Daemon/pull/409#discussion_r551039241)
> I don't how to unmark the PR as "changes requested", but I changed that.

You have to also click “resolve” on this thread.